src/components/userAuth/Logout.js

```diff
@@ -32,5 +33,17 @@ const Logout = () => {
     </Button>
   );
 };
+Logout.propTypes = {
+  apiUrl: PropTypes.string,
+  errorHandler: PropTypes.func,
+  submit: PropTypes.func,
+  navigate: PropTypes.func,
+  signOut: PropTypes.func,
+  status: PropTypes.string,
+  statusText: PropTypes.string,
+  user: PropTypes.object,
+  logout: PropTypes.func,
+  handleLogout: PropTypes.func,
+};
```

Comment:
Since this component is not expecting any props; we should not need to add propTypes here after all. For now at least :smile:

Reply:
thank you Akos ❤️ , commit 1bd5ab26f63ad7a21b158e0a26a812f1b339021e